sympy/assumptions/handlers/order.py

```diff
@@ -222,8 +231,16 @@ class AskPositiveHandler(CommonHandler):
 
     @staticmethod
     def Expr(expr, assumptions):
+        if expr.is_positive is None:
```

Comment:
I would try

```python
pos = expr.is_positive
return pos if pos is not None else fuzzy_and(
    (expr - a.args[0]).is_positive
    for a in conjuncts(assumptions))
```

Questions I have, however:

- should we consider other func's like Q.negative, Q.zero, Q.nonnegative, Q.nonpositive? Not knowing how these are stored I would try not to be too smart and just check them all as I did above
- should we use fuzzy_group as I suggest so if any result is False we can return False? Just because the first Q.positive assumption.func is not positive doesn't mean another might not resolve the query, does it?